source/guide/cordova-wrapper.md

```diff
@@ -1,19 +1,45 @@
 title: Cordova App Wrapper
 ---
-You can use Cordova to wrap your Quasar App into a native mobile App. We'll be using Quasar CLI to manage this.
+Cordova allows you to wrap your Quasar App into a native mobile App. In the following examples we'll be using Quasar CLI to manage this. After the initial setup you will need to wrap each project individually and add the desired platforms (Android and/or iOS).
 
-First install Cordova globally on your machine:
-``` bash
-$ npm install -g cordova
-```
 
-Then make sure you have the platform SDK installed on your machine, like [Android SDK](https://developer.android.com/studio/index.html#download).
-``` bash
-$ cordova requirements
-```
+## Initial setup steps
+
+There are some initial steps you need to take before you can wrap your apps using Cordova:
+
+1. First install Cordova globally on your machine:
+	``` bash
+	$ npm install -g cordova
+	```
+
+2. After this step you will need to install the Android platform SDK on your machine. You can [download the Android Studio here](https://developer.android.com/studio/index.html#download) and follow these [installation steps](https://developer.android.com/studio/install.html) afterwards.
+
+3. Update your ~/.bashrc file to contain the correct paths to your installation of Android Studio:
+
+	``` bash
+	export ANDROID_HOME="$HOME/Android/Sdk"
+	PATH=$PATH:$ANDROID_HOME/tools; PATH=$PATH:$ANDROID_HOME/platform-tools
+	```
+
+4. Start Android studio by changing into the folder you installed it in and run ```./studio.sh```. Next step is to install the individual SDKs:
+
+	Open the "Configure" menu at the bottom of the window:
+
+	![SDK manager](/images/Android-Studio-SDK-Menu.png "SDK manager")
+
+	Select the desired SDKs. As per May 2017 Cordova supports 4.4 and up and click on "Apply" to install the SDKs.
+
+	![SDK selection](/images/Android-Studio-SDK-selection.png "SDK selection")
 
-## Wrap your App
-Now let's create the Wrapper for your App.
+
+## Wrapping and configuration of your App
+
+Each Quasar project requires you to wrap your app initially. The following steps explain how to wrap your app and add the desired platforms.
```

Comment:
This sentence:

> Each Quasar project requires you to wrap your app initially.

is a little bit misleading...Maybe it's better to replace it with something like:

> To transform a Quasar project into a working Cordova app, you need to wrap it initially.